android/src/main/java/com/lynkxyz/noke/RNNokeModule.java

```diff
@@ -223,27 +222,28 @@ public void addNokeDevice(ReadableMap data, Promise promise) {
   }
 
   @ReactMethod
-  public void sendCommands(String command, Promise promise) {
+  public void sendCommands(ReadableMap data, Promise promise) {
```

Comment:
If we are breaking the compatibility on the interface itself, isn't it easier to just add two parameters instead of using a ReadableMap? Like, adding `mac` and `command` as parameters

Reply:
This is a good point. Using two separate parameters makes for a more descriptive function signature. I'll update.